packages/core/forms/src/composables/index.ts

```diff
@@ -0,0 +1,6 @@
+import useAbstractFields from './useAbstractFields'
+
+// All composables must be exported as part of the default object for Cypress test stubs
+export default {
+  useAbstractFields,
```

Comment:
suggestion: since this is the only composable, just import it directly where you need it instead of having this barrel file.

Reply:
We will eventually have a few others as well, I can still change the imports if that's preferred over the barrel file.